### PR TITLE
LPS-179736 Remove FF of LPS-171625 | LPS-182343 Update recipient placeholder

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issues:
https://issues.liferay.com/browse/LPS-179736;
https://issues.liferay.com/browse/LPS-182343.

### CODE REVIEW

**Comment by p-albuquerque** [comment]
the commit https://github.com/liferay-objects/liferay-portal/pull/2042/commits/7f0d96115f8fc42321f43ec8a26d59095004a0ff was already revisioned [here](https://github.com/liferay-objects/liferay-portal/pull/2036)

**Comment by marcelabc** [comment]
Hey @cgmonte, 
Could you help me to review this PR, please? :smile:

**Review by marcelabc — APPROVED** [review]
Backend part: LGTM 🎉

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx [hunk 1]
```diff
@@ -229,7 +229,7 @@ export function UserNotificationSettings({
 							Liferay.Language.get(
 								'use-terms-to-configure-recipients-for-this-notification-x'
 							),
-							'[%ENTRY_CREATOR%]',
+							'[%CURRENT_USER_ID%][%OBJECT_AUTHOR_ID%]',
```
Comment (cgmonte) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx:232]: Hey @Valmir-unicap (and also @p-albuquerque), The right placeholder is only `[%OBJECT_AUTHOR_ID%]`, not `[%CURRENT_USER_ID%][%OBJECT_AUTHOR_ID%]`. Please, fix it.

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hi @p-albuquerque, is this [bug](https://issues.liferay.com/browse/LPS-182343) fix necessary to remove the FF? If you remove this commit, which in the current state gives a wrong direction to the user, I can approve. Otherwise, it needs to be fixed before forwarding it.

**Comment by p-albuquerque** [comment]
> Hi @p-albuquerque, is this [bug](https://issues.liferay.com/browse/LPS-182343) fix necessary to remove the FF? If you remove this commit, which in the current state gives a wrong direction to the user, I can approve. Otherwise, it needs to be fixed before forwarding it.

The current placeholder info will became outdated, just when this FF is removed. Because the mentioned term will no longer exist. So I thought that would be interesting to forward both at same time, since they are small code adjusts.

PS: @Valmir-unicap already perform the changes requested and I append it in this commit.

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:forward:force

**Pull request closed without merging**